app/assets/stylesheets/tinder.scss

```diff
@@ -0,0 +1,112 @@
+#tinderslide {
+  width: 1000px;
+  height: 550px;
+  margin: auto;
+  background-color: $background;
+  margin-top: 100px;
+
+  img {
+    float: left;
+    margin: 20px 20px 0 20px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `20px 20px 0` instead of `20px 20px 0 20px`